Cargo.toml

```diff
@@ -1,5 +1,2 @@
 [workspace]
 members = ["rust/crd", "rust/operator-binary"]
-
-[patch."https://github.com/stackabletech/operator-rs.git"]
-stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feature/resource-quotas" }
```

Comment:
I would much rather have this just commented out. It's something that we need over and over again and I hate to google the exact syntax every time I need it. Especially the `//` hack is important. Have you seen it ? :)

Reply:
Wasn't aware this the `//` is part of a hack. Will re-add it.

Reply:
Re-added in faf573a